File: code-coaches/medium/carrot-cake/CarrotCake.cs
using System;

public class CarrotCake
{
    static string MakeCarrotCake(int num_carrots, int num_boxes)
    {
        int num_carrots_leftover = num_carrots % num_boxes;
        int num_carrots_to_make_a_cake = 7;

        if (num_carrots_leftover >= num_carrots_to_make_a_cake)
        {
            return "Cake Time";
        }
        else
        {
            int num_additional_carrots_need = num_carrots_to_make_a_cake - num_carrots_leftover;
            return $"I need to buy {num_additional_carrots_need} more";
        }
    }

    static void Main(string[] args)
    {
        int num_carrots = int.Parse(Console.ReadLine());
        int num_boxes = int.Parse(Console.ReadLine());
        Console.WriteLine(MakeCarrotCake(num_carrots, num_boxes));
    }
}

File: code-coaches/medium/carrot-cake/CarrotCake.swift
func makeCarrotCake(numCarrots: Int, numBoxes: Int) -> String {
    let numCarrotsLeftover = numCarrots % numBoxes
    let numCarrotsToMakeACake = 7

    if numCarrotsLeftover >= numCarrotsToMakeACake {
        return "Cake Time"
    } else {
        let numAdditionalCarrotsNeeded = numCarrotsToMakeACake - numCarrotsLeftover
        return "I need to buy \(numAdditionalCarrotsNeeded) more"
    }
}

func main() {
    let numCarrots = Int(readLine()!)!
    let numBoxes = Int(readLine()!)!
    print(makeCarrotCake(numCarrots: numCarrots, numBoxes: numBoxes))
}

main()
